src/docs/content/reference/current/core/injection/code/InjectionSampleKotlin.kt

```diff
@@ -39,6 +39,15 @@ setUp(
 )
 //#open-injection
 
+//#fractional-user
+//Injects 1 user every 2 seconds for 60 seconds
+setUp(
+  scn.injectOpen(
+    constantUsersPerSec(0.5).during(60)
+  ).protocols(httpProtocol)
+);
```

Comment:
no semi colon in Kotlin

Reply:
Do we want to add more code samples? Or just update the existing one with fractional values? https://github.com/gatling/gatling/blob/364da77b5702689a1cc685e50e1a5bf262ba84ae/src/docs/content/reference/current/core/injection/code/InjectionSampleJava.java#L37-L40